sql/scriptdev2/scriptdev2.sql

```diff
@@ -1799,6 +1799,18 @@ UPDATE creature_template SET ScriptName='npc_constrictor_tentacle' WHERE entry=3
 UPDATE creature_template SET ScriptName='npc_descent_madness' WHERE entry=34072;
 UPDATE creature_template SET ScriptName='npc_keeper_mimiron' WHERE entry=33412;
 UPDATE creature_template SET ScriptName='npc_keeper_thorim' WHERE entry=33413;
+-- Hodir
+UPDATE `creature_template` SET `SpellList`=3284500 WHERE `entry`=32845; -- Hodir (10)
+UPDATE `creature_template` SET `SpellList`=3284600 WHERE `entry`=32846; -- Hodir (25)
+-- Snowpacked Icicle Target
+UPDATE `creature_template` SET `StaticFlags1`=`StaticFlags1`|0x00000100|0x02000000 WHERE `entry`=33174;
+UPDATE `creature_template` SET `ScriptName`='' WHERE `entry`=33174;
+UPDATE `creature_template` SET `AIName`='', `ScriptName`='npc_snowpack_target' WHERE `Entry`=33174;
+UPDATE `creature_template` SET `AIName`='EventAI', `ScriptName` = '' WHERE `Entry` = 33169; -- Icicle
+-- Hodir's Helper NPCs
+UPDATE `creature_template` SET `StringId1`=6030701 WHERE `entry` IN (32941, 33333, 32950, 33332, 32946, 33331, 32948, 33330, 32901, 33325, 32900, 33328, 32893, 33327, 32897, 33326, 33342);
```

Comment:
only scriptname goes here. All the rest of the sql goes to db.

And if script is set, ai name is overriden so you dont need to do that.

Reply:
is there a reason why it's fragmented like this? I'll change it ofc, I'm just curious

Reply:
Yes, after a long discussion with cyberium we are trying to remove almost all sql from core, except script assignments to respective entries until a point in time when spell and other refactoring is reaching critical mass.